docs/resources/cdm_cluster.md

```diff
@@ -32,69 +35,112 @@ resource "huaweicloud_cdm_cluster" "cluster" {
+  provider-level region will be used. Changing this parameter will creates a new resource.
 
-* `availability_zone` - (Required, String, ForceNew) Available zone. Changing this parameter will create a new resource.
+* `name` - (Required, String, ForceNew) Specifies cluster name. Changing this parameter will create a new resource.
 
-* `flavor_id` - (Required, String, ForceNew) Flavor id. Changing this parameter will create a new resource.
+* `availability_zone` - (Required, String, ForceNew) Specifies available zone.
+  Changing this parameter will create a new resource.
 
-* `name` - (Required, String, ForceNew) Cluster name. Changing this parameter will create a new resource.
+* `flavor_id` - (Required, String, ForceNew) Specifies flavor id. Changing this parameter will create a new resource.
 
-* `security_group_id` - (Required, String, ForceNew) Security group ID. Changing this parameter will create a new
-  resource.
+* `version` - (Required, String, ForceNew) Specifies cluster version. Default value is `2.8.6.2`.
+ Changing this parameter will create a new resource.
 
-* `subnet_id` - (Required, String, ForceNew) Subnet ID. Changing this parameter will create a new resource.
+* `vpc_id` - (Required, String, ForceNew) Specifies VPC ID. Changing this parameter will create a new resource.
 
-* `version` - (Required, String, ForceNew) Cluster version. Changing this parameter will create a new resource.
+* `subnet_id` - (Required, String, ForceNew) Specifies subnet ID. Changing this parameter will create a new resource.
 
-* `vpc_id` - (Required, String, ForceNew) VPC ID. Changing this parameter will create a new resource.
+* `security_group_id` - (Required, String, ForceNew) Specifies security group ID.
+ Changing this parameter will create a new resource.
 
-* `email` - (Optional, List, ForceNew) Notification email addresses. The max number is 5. Changing this parameter will
-  create a new resource.
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies the enterprise project id.
+ Changing this parameter will create a new resource.
 
-* `enterprise_project_id` - (Optional, String, ForceNew) The enterprise project id. Changing this parameter will create
-  a new resource.
+* `is_auto_off` - (Optional, Bool, ForceNew) Specifies Whether to enable auto shutdown. The auto shutdown and scheduled
+ startup/shutdown functions cannot be enabled at the same time. When auto shutdown is enabled, if no job is running in
+  the cluster and no scheduled job is created, a cluster will be automatically shut down 15 minutes after it starts
+   running to reduce costs. Changing this parameter will create a new resource.
 
-* `is_auto_off` - (Optional, Bool, ForceNew) Whether to automatically shut down. Changing this parameter will create a
-  new resource.
+* `schedule_boot_time` - (Optional, String, ForceNew) Specifies time for scheduled startup of a CDM cluster.
```

Comment:
`schedule_boot_time` is not very good, but the published ones do not need to be updated.
It is recommended to add a description of the time format.